index.html

```diff
@@ -1,10 +1,44 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Lesson 8</title>
+        <link rel="icon" type="image/x-icon" href="https://cdn-icons-png.flaticon.com/512/119/119598.png">
+        <title>Lesson 7 - version 7.1</title>
         <link href="style.css" rel="stylesheet" type="text/css">
     </head>
     <body>
-        <h1>Lesson 8</h1>
+        <h1>Veronica Rubliuk</h1>
+        <img src="img.jpg" width="210" height="260" alt="lesson_7_robot.dreams_rub_ver">
+        <h3>junior front-end developer</h3>
+        <div>Work Expirience</div>
+        <br>
+        <table>
+            <thead>
+                <tr class="thHead">
```

Comment:
please, don't use capital letters in the name of classes

Reply:
okey

Reply:
done